docs/ref/modules/inventory-sync/flatbuffers.md

```diff
@@ -11,13 +11,16 @@ The inventory synchronization protocol is defined in flatbuffer schema files tha
 ```flatbuffers
 // Main message wrapper
 table Message {
-    content: MessageContent;
+    content: MessageType;
 }
 
-union MessageContent {
+union MessageType {
+    Data,
     Start,
-    Data, 
-    End
+    StartAck,
+    End,
+    EndAck,
+    ReqRet
```

Comment:
Although `StartAck` and `ReqRet` are mentioned in the union, the full definitions of these tables have not been added:
```
table StartAck {
    status: Status;
    session: ulong;
}

table ReqRet {
    seq: [Pair];
    session: ulong;
}

table Pair {
    begin: ulong;
    end: ulong;
}
```